clang/lib/Parse/ParseAST.cpp

```diff
@@ -168,7 +168,7 @@ void clang::ParseAST(Sema &S, bool PrintStats, bool SkipFunctionBodies) {
   for (Decl *D : S.WeakTopLevelDecls())
     Consumer->HandleTopLevelDecl(DeclGroupRef(D));
 
-  if (S.getLangOpts().SYCLIsDevice) {
+  if (S.getLangOpts().SYCLIsDevice || S.getLangOpts().SYCLIsHost) {
```

Comment:
What is the purpose of this change?
The code above was added to force emission of generated OpenCL kernel because it is not presented in the original source. I don't think that we need to solve this problem on host, since if I remember correct, we don't generate OpenCL kernel there.

Reply:
No, this is not needed. Removed host check.